src/UI/Buyer/src/app/wrapper-components/order-history-wrapper-component.ts

```diff
@@ -24,7 +24,7 @@ export class OrderHistoryWrapperComponent implements OnInit, OnDestroy {
   }
 
   setOrders = async (): Promise<void> => {
-    this.orders = await this.orderFilters.listOrders()
+    this.orders = await this.orderFilters.listOrders(window.location.pathname.toLowerCase().indexOf("quotes") != -1)
```

Comment:
can you assign this to a variable above just to make it more clear what its doing? Maybe something like `shouldListQuoteOrders`?

```javascript
const shouldListQuoteOrders = window.location.pathname.toLowerCase().indexOf("quotes") != -1;
this.orders = await this.orderFilters.listOrders(shouldListQuoteOrders )
```